db/queries/scores/queries.py

```diff
@@ -24,34 +24,45 @@ def get_scores_for_app_sub_crit(
     :return: dictionary.
     """
     if score_history:
-        stmt = select(Score).where(
-            Score.application_id == application_id,
-            Score.sub_criteria_id == sub_criteria_id
-            ).order_by(Score.date_created.desc())
+        stmt = (
+            select(Score)
+            .where(
+                Score.application_id == application_id,
+                Score.sub_criteria_id == sub_criteria_id,
+            )
+            .order_by(Score.date_created.desc())
+        )
```

Comment:
Could we move this part of the query into a variable and then just change the limit in the `else` case?

Reply:
Ah I've actually not looked at this code - it was just pre-commit formatting since whoever had committed it didn't run.  I'll look at adding your suggestion

Reply:
Refactored in https://github.com/communitiesuk/funding-service-design-assessment-store/pull/94/commits/0d51c337e6e310312c004c4ebd32774419efaafe